### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nikushaSheremet**
- [DEMO LINK](https://nikushaSheremet.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://nikushaSheremet.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="blocks">
+      <div class="blocks__block blocks__block--1">1</div>
+      <div class="blocks__block blocks__block--2">2</div>
+      <div class="blocks__block blocks__block--3">3</div>
+      <div class="blocks__block blocks__block--4">4</div>
+      <div class="blocks__block blocks__block--5">5</div>
+      <div class="blocks__block blocks__block--6">6</div>
```
Comment (carrotflower) [inline src/index.html:17]: delete blocks__block--1
delete blocks__block--2
etc
You don't need them if you do color with @each

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+    background-color: rgb(102, 0, 0);
+  }
+
+  &__block--5 {
+    background-color: rgb(51, 0, 0);
+  }
+
+  &__block--6 {
+    background-color: rgb(0, 0, 0);
+  }
```
Comment (carrotflower) [inline src/styles/main.scss:58]: You can do this with less code. Create variable and put your colors like this - &variable {
  rgb(255, 0, 0) 1,
  rgb(204, 0, 0) 2,
  rgb(153, 0, 0) 3,
  rgb(100, 0, 0) 4,
  rgb(51, 0, 0) 5,
  rgb(0, 0, 0) 6; 
  }

Then write it like this in your .block class
  @each $color, $order in $variable{
      &:nth-child(#{$order}) {
        background-color: $color;
      }

**Review by carrotflower — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (nikushaSheremet) [reply src/index.html]: fixed this. thank you for you comment!)

File: src/styles/main.scss (hunk 2 above)
Comment (nikushaSheremet) [reply src/styles/main.scss]: fixed)

**Review by carrotflower — APPROVED** [review]
Good job!

**Review by AndriiRut — APPROVED** [review]
🔥

**Pull request closed without merging**